lib/tapioca/gemfile.rb

```diff
@@ -30,9 +30,12 @@ def dependencies
       @dependencies ||= begin
         specs = definition.locked_gems.specs.to_a
 
+        # TODO: Somehow display these missing specs to the user?!
+        missing_specs = Array.new # rubocop:disable Style/EmptyLiteral
```

Comment:
Log them into a file and display the path to the log file in case of errors?

Reply:
Yeah, will probably be something like that. I added the TODO so that I don't have to think about that right now.

Reply:
Well, actually, I went ahead and refactored this so that I can print missing specs on the screen.